tests/console/commands/env/test_activate.py

```diff
@@ -77,6 +77,118 @@ def test_env_activate_prints_correct_script_on_windows(
     assert line == f'{prefix}"{tmp_venv.bin_dir / ext!s}"'
 
 
+@pytest.mark.parametrize(
+    "shell, ext, expected_prefix",
+    (
+        ("cmd", "activate.bat", ""),
+        ("pwsh", "activate.ps1", "& "),
+        ("powershell", "activate.ps1", "& "),
+    ),
+)
+@pytest.mark.skipif(not WINDOWS, reason="Only Windows shells")
+def test_env_activate_windows_shells_get_quoted_path_only(
```

Comment:
This is more or less just a copy of `test_env_activate_prints_correct_script_on_windows`, isn't it?